addons/bestja_organization/models.py

```diff
@@ -214,13 +214,20 @@ class UserWithOrganization(models.Model):
 
     coordinated_org = fields.One2many('organization', inverse_name='coordinator')
     organizations = fields.Many2many(
-        'res.users',
+        'organization',
         relation='organization_volunteers_rel',
         column1='volunteer',
         column2='organization',
         string="Organizacje"
     )
 
+    def __init__(self, pool, cr):
+        super(UserWithOrganization, self).__init__(pool, cr)
+        cls = type(self)
+        cls.FIELDS_ACCESS_ALL += ['coordinated_org']
+        cls.FIELDS_ACCESS_PRIVILEGED += ['organizations']
+        cls.FIELDS_ACCESS_OWNER += ['organizations']
```

Comment:
In the instance constructor we modify the class. If we initialize multiple instances of this class, the class will be modified multiple times, so the lists above will contain duplicates. Maybe we should use metaclass to monkey patch the class?